sdk/communication/azure-communication-rooms/azure/communication/rooms/_rooms_client.py

```diff
@@ -96,13 +99,15 @@ def create_room(
         *,
         valid_from: Optional[datetime] = None,
         valid_until: Optional[datetime] = None,
+        pstnDialOutEnabled: Optional[bool] = False,
```

Comment:
We would only use `Optional` here if the default was `None`.
```suggestion
        pstnDialOutEnabled: bool = False,
```